lib/encrubto/caesar/encryptor.rb

```diff
@@ -1,11 +1,10 @@
-
 module Encrubto::Caesar
     class Encryptor 
     
       ORIGINAL = 'abcdefghijklmnopqrstuvwxyz'
 
       def encrypt(str, offset)
-        if ( (str.is_a? String) && (offset.is_a? Integer) )
+        if (str.is_a?(String) && offset.is_a?(Integer))
```

Comment:
Remove the parentheses from the `if` statement
Expected:
`if str.is_a?(String) && offset.is_a?(Integer)`